backend/battle/urls.py

```diff
@@ -20,4 +21,9 @@
     path("battle/detail/<int:pk>/", BattleDetail.as_view(), name="battle_detail"),
     path('signup/', BattleSignUp.as_view(), name='signup'),
     path('signup/sucess', SignUpSucess.as_view(), name='signup_sucess'),
+    path(
+        'reset/<uidb64>/<token>/',
+        auth_views.PasswordResetConfirmView.as_view(
+            success_url=reverse_lazy(
+                'password_reset_complete')), name='password_reset_confirm'),
```

Comment:
Is this working? I don't see the password_reset_complete view/url here.

Reply:
It works

Reply:
but which page the users sees when password reset is confirmed?

Reply:
When you do that, the user will think they're resetting the password, when in fact they are setting it for the first time, so they'll be confused. You must redefine and have both behaviors: password reset and password set when first invited.

You can get the URLs and views from https://github.com/django/django/blob/1f4908b01aaf7b209521d40ca0f3eb48f2ef1988/django/contrib/auth/urls.py and re-implement as needed.

Reply:
Thank you, I'll make this improvement